incubator/command-dump/src/main/java/io/aklivity/zilla/runtime/command/dump/internal/airline/ZillaDumpCommand.java

```diff
@@ -275,13 +297,18 @@ public void run()
                 for (int i = 0; i < streamBufferCount; i++)
                 {
                     final RingBufferSpy streamBuffer = streamBuffers[i];
+                    MessagePredicate spyHandler = dumpHandlers[i]::handleFrame;
                     workCount += streamBuffer.spy(spyHandler, 1);
                 }
                 idleStrategy.idle(workCount);
             } while (workCount != exitWorkCount);
         }
-        catch (IOException ex)
+        catch (Exception ex)
         {
+            if (exceptions)
+            {
+                ex.printStackTrace();
```

Comment:
I added the `-e` switch for consistency with the `start` command. The engine is not started so it can't print exceptions, `ZillaMain` doesn't print the either, so I am printing them here,